storage/Filesystems/BlkFilesystem.h

```diff
@@ -46,13 +46,13 @@ namespace storage
 	static std::vector<BlkFilesystem*> get_all(Devicegraph* devicegraph);
 	static std::vector<const BlkFilesystem*> get_all(const Devicegraph* devicegraph);
 
-	virtual bool supports_label() const = 0;
-	virtual unsigned int max_labelsize() const = 0;
+	bool supports_label() const;
```

Comment:
BlkFilesystem is still an abstract class (since clone in Device is not defined). So you cannot call those functions on the base class (BlkFilesystem) but only the concrete derived classes (e.g. Ext4, Btrfs).